gemrb/core/Scriptable/Actor.cpp

```diff
@@ -8058,10 +8056,14 @@ void Actor::Draw(const Region &screen)
 			}
 		}
 
+		Game* game = core->GetGame();
+		ieDword flags = !ca->lockPalette ? BLIT_TINTED : 0;
+		game->ApplyGlobalTint(tint, flags);
+
```

Comment:
I guess I didn't test this after merge-conflict solving: In `subviews`, you put this into `DrawActorSprite`. But this too late as IV really affects more things that should be exempted from global tinting.

Reply:
What would be a practical example of that?

Reply:
Mirror-spells, invisibility